### PR TITLE
E 1397 add empty states

### PR DESCRIPTION
**Pull request by pixelfact**

### CODE REVIEW

File: app/u/[githubLogin]/features/total-earned-graph/total-earned-graph.tsx [hunk 1]
```diff
@@ -3,17 +3,36 @@ import { Money } from "utils/Money/Money";
...
+async function getStats(githubUserId: number) {
+  try {
+    return {
+      data: await usersApiClient.fetch.getUserPublicStats(githubUserId).request({
+        next: { revalidate: 120 },
+      }),
+      isError: false,
+    };
+  } catch (e) {
+    if ((e as FetchError).status === 404) {
```
Comment (haydencleary) [inline app/u/[githubLogin]/features/total-earned-graph/total-earned-graph.tsx:24]: Possible to make a `FetchError` class ? That way we could check if e is an instance of FetchError, might be better than casting.

File: app/u/[githubLogin]/features/languages-section/languages-section.tsx [hunk 2]
```diff
@@ -39,7 +42,11 @@ export async function LanguagesSection(props: TLanguagesSection.Props) {
         <DetailsAccordion details={languages} githubUserId={githubUserId} />
       ) : (
         <Card background={"base"}>
-          <Typography variant="body-m" translate={{ token: "v2.pages.publicProfile.languagesDetails.empty" }} />
```
Comment (haydencleary) [inline app/u/[githubLogin]/features/languages-section/languages-section.tsx:42]: Did you remove this translation ?

Comment (pixelfact) [reply app/u/[githubLogin]/features/languages-section/languages-section.tsx:42]: remove translation

Comment (pixelfact) [reply app/u/[githubLogin]/features/languages-section/languages-section.tsx:42]: done

**Review by haydencleary — APPROVED** [review]

**Pull request merged**